src/Yoti/Util/Profile/AnchorConverter.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+namespace Yoti\Util\Profile;
+
+use Traversable;
+use phpseclib\File\ASN1;
+use phpseclib\File\X509;
+use Attrpubapi_v1\Anchor;
+use Yoti\Entity\Anchor as YotiAnchor;
+
+class AnchorConverter
+{
+    /**
+     * Convert Protobuf Anchor to a map of Yoti Anchor
```

Comment:
Think this could be worded a bit differently, maybe:
`Convert Protobuf Anchor to a map of oid -> Yoti Anchor` ?

Reply:
I am not sure a bout using a long name for a function.

Reply:
I was referring to the doc comments 👍

Reply:
Ok sure.